graylog2-web-interface/src/views/components/searchbar/queryvalidation/validateQuery.ts

```diff
@@ -38,7 +38,7 @@ export const validateQuery = (
   { queryString, timeRange, streams, streamCategories, filter, ...rest }: ValidationQuery,
   userTimezone: string,
 ): Promise<QueryValidationState> => {
-  if (!queryExists(queryString) && !queryExists(filter) && !timeRange && streams?.length === 0) {
+  if (!queryExists(queryString) && !queryExists(filter) && !timeRange && (streams ? streams.length === 0 : true)) {
```

Comment:
A common idiom would be to express it like this:

```suggestion
  if (!queryExists(queryString) && !queryExists(filter) && !timeRange && !streams?.length) {
```